tests/ios/frameworks/strict-deps/a/BUILD.bazel

```diff
@@ -3,6 +3,7 @@ load("//rules:framework.bzl", "apple_framework")
 apple_framework(
     name = "a",
     srcs = glob(["*.swift"]),
+    infoplists = ["//rules/test_host_app:Info.plist"],
```

Comment:
does this test the new code? Aka if we add this line without all other work, will we still get a green build (just no op?)

Reply:
It will fail with the error
```
ERROR: /Users/cshi/Development/rules_ios/tests/ios/frameworks/strict-deps/a/BUILD.bazel:3:16: //tests/ios/frameworks/strict-deps/a:a_swift: no such attribute 'infoplists' in 'swift_library' rule
ERROR: /Users/cshi/Development/rules_ios/tests/ios/frameworks/strict-deps/a/BUILD.bazel:3:16: //tests/ios/frameworks/strict-deps/a:a_objc: no such attribute 'infoplists' in 'objc_library' rule
ERROR: error loading package 'tests/ios/frameworks/strict-deps/a': Package 'tests/ios/frameworks/strict-deps/a' contains errors
```